web_translate_dialog/static/src/js/web_translate_dialog.js

```diff
@@ -111,9 +111,12 @@ var TranslateDialog = Dialog.extend({
     set_maxlength: function(){
         // set maxlength if initial field has size attr
         _.each(this.translatable_fields, function(field_name){
-            var size = $('[name='+field_name+']')[0].maxLength;
-            if (size > 0){
-                this.$('input.oe_translation_field[name$="'+field_name+'"], textarea.oe_translation_field[name$="'+field_name+'"]').attr('maxlength', size);
+            var field = $('[name='+field_name+']');
+            if (field.length > 0) {
```

Comment:
question: how comes that a translatable field is not there? If it's true, then something in `translatable_fields` or `get_translatable_fields` might be broken? :thinking: